x-pack/plugins/monitoring_collection/server/plugin.ts

```diff
@@ -85,7 +96,46 @@ export class MonitoringCollectionPlugin implements Plugin<MonitoringCollectionSe
     };
   }
 
+  private configureOpentelemetryMetrics() {
+    const meterProvider = new MeterProvider({
+      resource: new Resource({
+        [SemanticResourceAttributes.SERVICE_NAME]: 'kibana',
```

Comment:
We may want the service name to ensure uniqueness even when the observability backend receives signals from multiple kibana instances.

Reply:
good point! A configurable override makes sense to me here.

For the multiple backend instances, we may want to leverage `service.node.role` which I added in https://github.com/elastic/ecs/pull/1916

Reply:
I don't _think_ I see any similar field on https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-semantic-conventions/src/resource/SemanticResourceAttributes.ts so probably more reason to consider more direct ECS & otel work in the future :)

Reply:
@elastic/kibana-core is there a way I could reference `server.name` for this?

Reply:
Yea, you can access the server's name via `httpSetup.getServerInfo().name` or `httpStart.getServerInfo().name`

Reply:
You could also maybe use the Kibana instance's UUID, via `PluginInitializerContext.env.instanceUuid`

Reply:
I think that could be a great fit for `service.instance.id` I'll look at wiring up some of the other SemanticResourceAttributes as well.